src/staticobject.h

```diff
@@ -81,13 +76,28 @@ class StaticObjectList
 	void serialize(std::ostream &os);
 	void deSerialize(std::istream &is);
 
+	// Never permit to modify outside of here. Only this object is responsible of m_stored and m_active modifications
+	const std::vector<StaticObject>& getAllStored() const { return m_stored; }
+	const std::map<u16, StaticObject> &getAllActives() const { return m_active; }
+
+	inline void setActive(u16 id, const StaticObject &obj) { m_active[id] = obj; }
+	inline size_t getActiveSize() const { return m_active.size(); }
+	inline void clearActives() { m_active.clear(); }
+	inline size_t getStoredSize() const { return m_stored.size(); }
+	inline void clearStored() { m_stored.clear(); }
+	inline bool isStoredEmpty() const { return m_stored.empty(); }
```

Comment:
the const methods here can all be done via `getAllStores` or `getAllActives`, so not necessary